tests/test_e2e/test_pebble.py

```diff
@@ -365,3 +365,23 @@ def test_exec_wait_output_error(charm_cls):
         proc = container.exec(["foo"])
         with pytest.raises(ExecError):
             proc.wait_output()
+
+
+def test_pebble_custom_notice(charm_cls):
```

Comment:
Unless I'm missing something, it looks like we're testing `get_notices` here, but not that the actual event and `event.notice` object is what we expect. Should we have a charm that actually does/sets something on `_on_pebble_custom_event` so we can assert the event and event.notice looks good?

Reply:
Yeah, fair call, will do.